testkit/src/main/java/akka/kafka/testkit/javadsl/BaseKafkaTest.java

```diff
@@ -43,11 +43,8 @@
 
   public final Logger log = LoggerFactory.getLogger(getClass());
 
-  protected final Materializer materializer;
-
-  protected BaseKafkaTest(ActorSystem system, Materializer materializer, String bootstrapServers) {
```

Comment:
It was easy to drop all these references, but I wonder if it is too disruptive for end users since the constructors for this and other testkit superclasses were never deprecated before 2.1.0.

Reply:
/cc @ennru

Reply:
We should keep a backwards-compatible constructor around.
Let's enable Mima for the testkit to make conscious decisions about these things.

In the new variant `system` should be `ClassicActorSystemProvider`.

Reply:
:+1: